domaintools/api.py

```diff
@@ -534,6 +534,8 @@ def iris_detect_manage_watchlist_domains(self, watchlist_domain_ids, state, **kw
 
         state: str: required. Valid values are: ["watched", "ignored"]
         """
+        if isinstance(watchlist_domain_ids, str):
+            watchlist_domain_ids = watchlist_domain_ids.split(',')
```

Comment:
From the command line there is no way to pass a list so we need to be able to make it a list.